app.js

```diff
@@ -0,0 +1,245 @@
+          "Successfully won Techcrunch Disrupt",
+          "Optimized an algorithm that holds the current world record for Weisman Scores"
+        ]
+      }
+    ],
+    "volunteer": [
+      {
+        "organization": "CoderDojo",
+        "position": "Teacher",
+        "url": "http://coderdojo.example.com/",
+        "startDate": "2012-01-01",
+        "endDate": "2013-01-01",
+        "summary": "Global movement of free coding clubs for young people.",
+        "highlights": [
+          "Awarded 'Teacher of the Month'"
+        ]
+      }
+    ],
+    "education": [
+      {
+        "institution": "University of Oklahoma",
+        "area": "Information Technology",
+        "studyType": "Bachelor",
+        "startDate": "2011-06-01",
+        "endDate": "2014-01-01",
+        "gpa": "4.0",
+        "courses": [
+          "DB1101 - Basic SQL",
+          "CS2011 - Java Introduction"
+        ]
+      }
+    ],
+    "awards": [
+      {
+        "title": "Digital Compression Pioneer Award",
+        "date": "2014-11-01",
+        "awarder": "Techcrunch",
+        "summary": "There is no spoon."
+      }
+    ],
+    "publications": [
+      {
+        "name": "Video compression for 3d media",
+        "publisher": "Hooli",
+        "releaseDate": "2014-10-01",
+        "url": "http://en.wikipedia.org/wiki/Silicon_Valley_(TV_series)",
+        "summary": "Innovative middle-out compression algorithm that changes the way we store data."
+      }
+    ],
+    "skills": [
+      {
+        "name": "Web Development",
+        "level": "Master",
+        "keywords": [
+          "HTML",
+          "CSS",
+          "Javascript"
+        ]
+      },
+      {
+        "name": "Compression",
+        "level": "Master",
+        "keywords": [
+          "Mpeg",
+          "MP4",
+          "GIF"
+        ]
+      }
+    ],
+    "languages": [
+      {
+        "language": "English",
+        "fluency": "Native speaker"
+      }
+    ],
+    "interests": [
+      {
+        "name": "Wildlife",
+        "keywords": [
+          "Ferrets",
+          "Unicorns"
+        ]
+      }
+    ],
+    "projects": [
+      {
+        "name": "Miss Direction",
+        "description": "A mapping engine that misguides you",
+        "highlights": [ 
+          "Won award at AIHacks 2016",
+          "Built by all women team of newbie programmers",
+          "Using modern technologies such as GoogleMaps, Chrome Extension and Javascript"
+        ],
+        "keywords": [
+          "GoogleMaps", "Chrome Extension", "Javascript"
+        ],
+        "startDate": "2016-08-24",
+        "endDate": "2016-08-24", 
+        "url": "missdirection.example.com",
+        "roles": [ 
+          "Team lead", "Designer"
+        ],
+        "entity": "Smoogle",
+        "type": "application"
+      }
+    ]
+  }
+
+
+
+function getResume(){
+    let element
+    let className
+    
+    var basics = document.getElementById("basics")

```

Comment:
The mix of ES6 and ES5 syntax is a bit confusing. Sticking to one or the other would be good for consistency :)